compiler/noirc_driver/Cargo.toml

```
@@ -26,3 +26,4 @@ rust-embed.workspace = true
 tracing.workspace = true
 
 aztec_macros = { path = "../../aztec_macros" }
+num-traits = "0.2.19"
```

```suggestion
num-traits.workspace = true
```

We've defined a version to use across the workspace in the root `Cargo.toml`. Can you replace all the other instances with this?